wdlc.sh.in

```diff
@@ -598,8 +598,6 @@ if [ -d "${intermediate_output_path}" ]; then
     usage 1 "${wdlc_datadir}"
 fi
 
-protoc_search_path_options="--proto_path ${wdlc_includedir}"
```

Comment:
Likely a vacuous comment given the behavior in shell, but I would feel better if the protoc_search_path_options was initialized to empty